src/support.cpp

```diff
@@ -1314,16 +1295,8 @@ std::pair<Polygons, Polygons> AreaSupport::computeBasicAndFullOverhang(const Sli
     //     Polygons overhang =  basic_overhang.unionPolygons(support_extension);
     //         presumably the computation above is slower than the one below
 
-    {
-        const auto nozzle_diameter = mesh.settings.get<coord_t>("machine_nozzle_size");
-        const coord_t min_circumference = nozzle_diameter * M_PI;
-        const double min_area = INT2MM2((nozzle_diameter * nozzle_diameter) / 4 * M_PI);
-        constexpr bool remove_holes = true;
-        basic_overhang.removeSmallAreaCircumference(min_area, min_circumference, remove_holes);
-    }
-
-    Polygons overhang_extended = basic_overhang.offset(max_dist_from_lower_layer + MM2INT(0.1)); // +0.1mm for easier joining with support from layer above
-    Polygons full_overhang = overhang_extended.intersection(supportLayer_supportee);
+    Polygons overhang_extented = basic_overhang.offset(max_dist_from_lower_layer + MM2INT(0.1)); // +0.1mm for easier joining with support from layer above
```

Comment:
Can you remove the ancient commented out code above, even though it isn't part of your changes?